modules/libcom/src/osi/os/RTEMS-posix/devLibVMEOSD.c

```diff
@@ -0,0 +1,6 @@
+#ifdef RTEMS_LEGACY_STACK
+#include "../RTEMS-kernel/devLibVMEOSD.c"
+#else
+#pragma message "\n VME-Support only with RTEMS Legacy stack\n" 
```

Comment:
This statement confuses me.  Maybe I don't understand the full scope of what is in `libbsd.a`.  I thought this was only network stuff, include NIC drivers?  This comment implies that some PCI bus infrastructure is there as well.

Reply:
I'm not 100% sure either, but I found this:
https://devel.rtems.org/browser/rtems-libbsd/libbsd.txt

"The libbsd makes FreeBSD subsystems like TCP/IP, USB, SD and some more usable for RTEMS."

Reply:
Should we ask the RTEMS developers whether/how they (plan to) support PCI and VME bus interfaces? For most applications that pick a board with an interface to it, VMEbus access is going to be an essential part of that application. This seems like a question for the RTEMS devel mailing list (which I don't follow) to get their take on what we should do.

Note that for targets that don't have a VME interface, the devLibVMEOSD.c file still needs to be compiled and define this dummy:
```C
/* No known VME interface here, provide a dummy */
devLibVME *pdevLibVME;
```
That makes it possible for IOC applications to be built for all targets without having to exclude the ones that don't have VME. Any attempt to access the VMEbus on such a target will fail at runtime though, unless the application also provides another implementation and sets that pointer (PCI to VME adaptors exist, and some EPICS sites use them via this mechanism).